app/routes/missions/swift.mdx

```diff
@@ -276,6 +276,24 @@ https://swift.gsfc.nasa.gov/archive/
   </table>
 </div>
 
+**JSON-Serialized GCN Notice Types in GCN Kafka:**
+
+The GUANO system distributes alerts and localizations for sub-threshold detections of gamma-ray transients around the times of other triggers.
+These are bursts that did not result in any of the automatic GCN Notices described above, and is the result of analysis after the data arrives on the ground.
+
+These notices are published on the GCN Kafka topic **swift.bat.guano**.
```

Comment:
```suggestion
These notices are published on the GCN Kafka topic `swift.bat.guano`.
```

We should probably change that topic name to `gcn.notices.swift.bat.guano`, right @Tohuvavohu, @jracusin, @Vidushi-GitHub, @jracusin?

Reply:
when i successfuly push/listen to notices i do 
`consumer.subscribe(['swift.bat.guano'])`

Reply:
Yes, that's right, @Tohuvavohu. However, it seems like we are moving in the direction of the `gcn.notices.*` style. That's where we have placed Glowbug. Would it be OK with you if we move GUANO notices to `gcn.notices.swift.bat.guano`?